site/content/en/docs/manual/advanced/slice-and-join.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Join and slice tools'
+linkTitle: 'Join and slice tools'
+weight: 18
+description: 'This section explains how to slice or join several labels'
+---
+
+In CVAT you can modify shapes by either joining multiple shapes into
+a single label or slicing a single label into several shapes.
+
+This document provides guidance on how to perform these operations effectively.
+
+See:
+
+- [Joining CVAT labels](#joining-cvat-labels)
+- [Slicing CVAT labels](#slicing-cvat-labels)
+
+## Joining CVAT labels
+
+The **Join masks** tool (![Join masks tool icon](/images/join-masks-icon.jpg)),
+is specifically designed to work with mask annotations.
+
+This tool is useful in scenarios where a single object
+in an image is annotated with multiple shapes,
+and there is a need to merge these shapes into a single, unified shape.
+
+![Join masks](/images/joining-tool-01.jpg)
+
+To join masks, do the following:
+
+1. From the [**Edit**](/docs/manual/basics/controls-sidebar/#edit) block,
+   select **Join masks** ![Join masks tool icon](/images/join-masks-icon.jpg).
+2. Click on the labels, to select masks that you want to join.
+3. (Optional) To remove the selection click one more time on the mask.
+4. Click again on **Join masks**![Join masks tool icon](/images/join-masks-icon.jpg)
+   (**J**) to execute the join operation.
+
+Upon completion, the selected masks will be joined into a single mask.
+
+![Join masks gif](/images/joining-tool-02.gif)
+
+## Slicing CVAT labels
+
+The **Slice mask/polygon** (![Slicing tool icon](/images/slicing-tool-icon.jpg))
+is compatible with both mask and polygon annotations.
+
+This tool is useful in scenarios where multiple objects in an image
+are annotated with one shape,
+and there is a need to slice this shape into multiple parts.
```

Comment:
Maybe we should add, that a shape can be sliced only in two parts in time.
Apply the slice tool several times to split a shape to as many parts as you need.